content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,468 @@
+
+.. _italy/internal-reverse:
+
+Internal reverse charge
+-----------------------
+
+.. important::
+   Odoo currently does not support **internal reverse charge** processes.
+
+In some cases, vendor bills have the VAT duty **reverse charged** on the *buyer* for **national
+purchases** too (e.g., cleaning company invoices, construction sub-contractor invoices, scrap
+selling, tablets and mobile phones, etc.). For these operations, the *seller* or *provider* issues
+an electronic invoice with :guilabel:`Tipo Documento` code `TD01`, `TD02` for down payments, or
+`TD04` for credit notes, with the **tax-exemption code** ranging from `N6.1` to `N6.9`.
+
+The *buyer* is required to issue themselves an invoice (i.e., **self-billing**) for the VAT due with
+:guilabel:`Tipo Documento` code `TD16`, which is sent through the :abbr:`SdI (Systema di
+Interscambio)` with the buyer *themselves* as **recipient** (using their own :guilabel:`Codice
+Destinatario`). That way, they can deduct their VAT. The **self-billed** invoice must be issued and
+recorded in the VAT sales register within **15 days** from receipt.
+
+.. _italy/grids:
+
+'Reverse Charge' tax grids
+--------------------------
+
+The Italian localization has a specific **tax grid** section for **reverse charge** taxes. These
+tax grids are identifiable by the :ref:`VJ <italy/grids>` tag, and can be found under
+:menuselection:`Accounting --> Reporting --> Audit Reports: Tax Report`.
+
+.. image:: italy/italy-grids.png
    :align: center
-   :alt: Electronic invoicing status (waiting for confirmation)
+   :alt: Italian reverse charge tax grids
+
+San Marino
+==========
+
+Invoices
+--------
+
+San Marino and Italy have special agreements on e-Invoicing operations. As such, **invoices** follow
+the regular **reverse charge** rules. Additional requirements are not enforced by Odoo, however, the
+user is requested by the **State** to:
+
+- Select a tax with the option :guilabel:`Has exoneration of tax (Italy)` ticked, and the
+  :guilabel:`Exoneration` set to `N3.3`;
+- Use the generic :abbr:`SdI (Sistema di Interscambio)` :guilabel:`Codice Destinatario` `2R4GT08`.
+  The invoice is then routed by a dedicated office in San Marino to the correct business.
+
+Bills
+-----
+
+When a **paper bill** is received from San Marino, any Italian company **must** submit that invoice
+to the **Agenzia delle Estrate** by indicating the e-Invoice's :guilabel:`Typo Documento` field with
+the special value `TD28`.
+
+.. tabs::
+
+   .. tab:: `TD28`
+
+      Odoo exports a move as `TD28` if the following conditions are met:
+
+      - Is a vendor bill;
+      - At least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+      - The **country** of the partner is **San Marino**.
+
+Pubblica amministrazione (B2G)
+==============================
+
+Digital qualified signature
+---------------------------
+
+.. note::
+   Odoo does not support the use of **digital qualified signature** for now.
```

Comment:
**Warning** This is important.

We don't support the WHOLE Public Administration flow. Not even sending simple XMLs to them, we block it. 
The problem is not just with the Qualified Signature. Also CIG and CUP etc. are not supported, or anything else that might be needed. 
The user may think: "If it's just the signature, I'll provide it myself" but then we don't handle all the rest of the flow and the information necessary, and he will make a mess.

To keep it simple, please put as a warning with the text like:
"We don't support e-invoicing to **any** partner belonging to the Italian Public Administration. All the information below is just a reference for future implementations."

It must be a warning on the entire Public Administration section and not a note. Like this:
![image](https://user-images.githubusercontent.com/1665365/215101408-b0f96e38-3578-446f-a501-350495920e52.png)